instructor-guides/advanced_data_protection_INSTRUCTOR_GUIDE.md

```diff
@@ -0,0 +1,123 @@
+
+As mentioned in the Prerequisites section, all HashiCorp instructors and TAs for workshops should register with Instruqt and then post a message in our Slack channel, #proj-instruqt, asking to be added to the HashiCorp organization in Instruqt.
+
+Members of the HashiCorp organization in Instruqt can hover over any challenge in any track in that organization and see a "Skip to Challenge" button. After starting a track, you can use these buttons to run the track's setup and solve scripts up to the challenge you want to skip to. When skipping to a challenge, always be sure to click the "Normal Skip" button too.
+
+https://play.instruqt.com/hashicorp/tracks/vault-basics
+
+Go through this track start to finish and make sure you understand all the challenges. This track covers the Vault CLI and UI, running a dev Vault server and writing a secret to it, using the Vault HTTP API, running a production Vault server, using the KVv2 secrets engine and the Userpass auth method, and Vault policies.
+
+https://play.instruqt.com/hashicorp/tracks/vault-dynamic-database-credentials
+
+Go through this track start to finish and make sure you understand all the challenges. This track covers Vault's database secrets engine.
+
+Go through this track start to finish and make sure you understand all the challenges. This track covers Vault's encryption as a service capabilities as implemented in its Transit and Transform Secret Engines.
+
+The following track is optional and focuses on Tokenization.
+
+https://play.instruqt.com/hashicorp/tracks/vault-advanced-data-protection-with-tokenization
+
+Go through this track start to finish and make sure you understand all teh challenges. This track covers Vault's KMIP secret engine.
+
+https://play.instruqt.com/hashicorp/tracks/vault-kmip-secrets-engine-mongodb
+
+
+### Configuring the Instruqt Pools
+We recommend that you configure Instruqt pools for each Instruqt track used in this workshop 1 hour before you plan to use the track. Please see this Confluence [doc](https://hashicorp.atlassian.net/wiki/spaces/SE/pages/511574174/Instruqt+and+Remark+Contributor+Guide#InstruqtandRemarkContributorGuide-ConfiguringInstruqtPools) for instructions.
+
+### Timing
+The following schedule assumes you have a group of participants who are brand new to Vault. You should budget about four hours for this workshop including a short break. (But in virtual workshops, participants are expected to take breaks during the labs.)
+
+This is meant as a guideline, you can adjust as needed. If you prefer to alternate the slides of the "Interacting with Vault" chapter with the first 3 challenges of the "Vault Basics" track, you can do that.
+
+If you want to cover the optional AWS slides and Instruqt tracks, add 1 hour.
+
+0:00 - 0:10 - Wait for attendees, make introductions<br>
```

Comment:
Please adjust this section to use topics that correspond to your ADP version of the workshop and suitable timings